covid_graphs/setup.py

```diff
@@ -21,10 +21,13 @@
     dependency_links=[],
     python_requires=">=3.7",
     packages=find_namespace_packages(),
-    scripts=["covid_graphs/scripts/prepare_data.py", "covid_graphs/scripts/scatter_plot.py"],
     entry_points={
         "console_scripts": [
-            "covid_graphs.run_server=covid_graphs.scripts.server:run_server"
+            "covid_graphs.run_server = covid_graphs.scripts.server:run_server",
+            "covid_graphs.country = covid_graphs.country:main",
+            "covid_graphs.heat_map = covid_graphs.heat_map:main",
+            "covid_graphs.scatter_plot = covid_graphs.scatter_plot:main",
```

Comment:
I'd suggest commands that are a bit more verbose, e.g. sth like:
```
covid_graphs.create_country_plot
covid_graphs.create_heat_map
covid_graphs.create_scatter_plot
```
(Maybe `show` instead of `create`.)

Reply:
In principle, but these are command line tools and typing all of them is a hassle, plus they take too much space on the command line. I suggest shortening the prefix, as it doesn't carry that much information. For example:
```suggestion
            "cg.scatter_plot = covid_graphs.scatter_plot:main",
```

Reply:
I disagree, I think clarity trumps convenience here. It is actually not bad at all - the command autocompletes on tabs, and usually at least I type the command once and reuse over again.

Reply:
Also it is a suggestion, feel free to go with what you think is best.

Reply:
Also a small note, for `run_server` I enabled short versions of the params, `-d`, `-p` and `-e`.

Reply:
My autocomplete doesn't work and I actually have a pretty good setup. When I type the command once, I get auto suggestions though:
![Screenshot_20200414_132212](https://user-images.githubusercontent.com/3822689/79219671-f9b49180-7e52-11ea-9c6e-ebaee40bfdb6.png)

I'll think about it for a few more hours, but I think I'll go with the shortened prefix.

Reply:
After some thought I went with your 'show*' suggestions.